autoPyTorch/api/base_task.py

```diff
@@ -1244,17 +1242,37 @@ def __del__(self) -> None:
         if hasattr(self, '_backend'):
             self._backend.context.delete_directories(force=False)
 
-    @typing.no_type_check
     def get_incumbent_results(
-            self
-    ):
-        pass
+        self,
+        include_traditional: bool = False
+    ) -> Tuple[Configuration, Dict[str, Union[int, str, float]]]:
+        """
+        Get Incumbent config and the corresponding results
+        Args:
+            include_traditional: Whether to include results from tradtional pipelines
 
-    @typing.no_type_check
-    def get_incumbent_config(
-            self
-    ):
-        pass
+        Returns:
+
+        """
+        assert self.run_history is not None, "No Run History found, search has not been called."
+        if self.run_history.empty():
```

Comment:
maybe you can add the above assert here, like:
```
if self.run_history is none or self.run_history.empty():
```

Reply:
doing so causes mypy to not understand that we checked for run_history is not None. I get `"None" has no attribute "<attr>"` for various statements in this function

Reply:
Oh, it was because of mypy, ok ok makes sense